persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAO.kt

```diff
@@ -37,7 +37,6 @@ interface MessageDAO {
      * @see insertOrIgnoreMessage
      */
     suspend fun insertOrIgnoreMessages(messages: List<MessageEntity>)
-    fun needsToBeNotified(id: String, conversationId: QualifiedIDEntity): Boolean
```

Comment:
I _think_ this exists because of tests. 

Like, there're some tests that test only this, instead of testing what `insertOrIgnoreMessage` does.

Ideally this should be changed to do the proper thing, but I wouldn't worry doing this in this PR.

Maybe adding a TODO and we can worry about this later so we don't hold back this PR is sufficient for now.